### DISCUSSION TITLE
Createsessionproxy() 사용 방법

### DISCUSSION THREAD

**Discussion by iwog3927** [storycraft/node-kakao#323] [Q&A]
안녕하세요.

클라이언트에 proxy 적용 방법을 문의드립니다.

pc 카카오톡에 proxy 적용 기능이 있고 Talkclient에 createsessionproxy()라는 메소드가 있는걸보면 노드카카오로도 프록시 적용이 가능할거 같은데 사용법을 모르겠네요.    혹시 가능하다면 간단한 프록시 적용 예제 방법 알려주시면 감사하겠습니다.

감사합니다.

**Comment by storycraft** [comment]
createSessionProxy는 해당 TalkClient의 현재 세션을 사용할수 있게 해주는 프록시 객체 생성 메서드입니다.

현재 NodeKakao에서는 프록시 관련 기능을 제공하고 있지 않습니다. 외부 프로그램이나 라이브러리를 사용해주세요
